<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequence Face Landmarks Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="display">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="frame_layout">
        <item>
         <widget class="QLabel" name="curr_frame_lbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>------</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="frame_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="max_frame_lbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>------</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttons_layout">
        <item>
         <widget class="QToolButton" name="backward_btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="play_pause_btn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="forward_btn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>8</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="hierarchy_lbl">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>:/images/hierarchy.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="maximum">
           <number>19</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>8</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="face_lbl">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="pixmap">
           <pixmap>:/images/face.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QToolButton" name="keyframe_btn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="actionOpen"/>
    <addaction name="action_Save"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionContours"/>
    <addaction name="actionBorders"/>
    <addaction name="actionSegmentation"/>
    <addaction name="actionPostprocessing"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>&amp;Playback</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionBackward"/>
    <addaction name="actionForward"/>
   </widget>
   <widget class="QMenu" name="menuPostprocessing">
    <property name="title">
     <string>&amp;Postprocessing</string>
    </property>
    <addaction name="actionDisconnected"/>
    <addaction name="actionHoles"/>
    <addaction name="actionSmooth"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEditSegmentation"/>
    <addaction name="actionEditContours"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_View"/>
   <addaction name="menuPostprocessing"/>
   <addaction name="menuPlayback"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="actionOpen"/>
   <addaction name="action_Save"/>
   <addaction name="actionSaveAs"/>
  </widget>
  <widget class="QToolBar" name="viewToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionContours"/>
   <addaction name="actionBorders"/>
   <addaction name="actionSegmentation"/>
   <addaction name="actionPostprocessing"/>
  </widget>
  <widget class="QToolBar" name="postToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDisconnected"/>
   <addaction name="actionHoles"/>
   <addaction name="actionSmooth"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionPlay">
   <property name="icon">
    <iconset theme=":/images/play.png"/>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionBackward">
   <property name="icon">
    <iconset theme=":/images/backward.png"/>
   </property>
   <property name="text">
    <string>&amp;Backward</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset theme=":/images/forward.png"/>
   </property>
   <property name="text">
    <string>&amp;Forward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionKeyframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme=":/images/key.png"/>
   </property>
   <property name="text">
    <string>Keyframe</string>
   </property>
   <property name="toolTip">
    <string>Toggle keyframe</string>
   </property>
   <property name="statusTip">
    <string>Toggle keyframe</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme=":/images/open.png"/>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open an existing file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset theme=":/images/new.png"/>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset theme=":/images/save.png"/>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the document to disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme=":/images/save-as.png"/>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save the document under a new name</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme=":/images/close.png"/>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
   <property name="statusTip">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme=":/images/exit.png"/>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionContours">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme=":/images/contours.png"/>
   </property>
   <property name="text">
    <string>&amp;Contours</string>
   </property>
   <property name="toolTip">
    <string>Show Contours</string>
   </property>
  </action>
  <action name="actionBorders">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme=":/images/borders.png"/>
   </property>
   <property name="text">
    <string>&amp;Borders</string>
   </property>
   <property name="toolTip">
    <string>Show Borders</string>
   </property>
  </action>
  <action name="actionSegmentation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme=":/images/segmentation.png"/>
   </property>
   <property name="text">
    <string>&amp;Segmentation</string>
   </property>
   <property name="toolTip">
    <string>Show Segmentation</string>
   </property>
  </action>
  <action name="actionPostprocessing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme=":/images/postprocess.png"/>
   </property>
   <property name="text">
    <string>&amp;Postprocessing</string>
   </property>
   <property name="toolTip">
    <string>Show Postprocessing</string>
   </property>
  </action>
  <action name="actionDisconnected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme=":/images/postprocess_disconnected.png"/>
   </property>
   <property name="text">
    <string>Disconnected</string>
   </property>
   <property name="toolTip">
    <string>Remove smaller disconnected segmented pixel components</string>
   </property>
   <property name="statusTip">
    <string>Remove smaller disconnected segmented pixel components</string>
   </property>
  </action>
  <action name="actionHoles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme=":/images/postprocess_holes.png"/>
   </property>
   <property name="text">
    <string>Holes</string>
   </property>
   <property name="toolTip">
    <string>Fill holes in the segmentation</string>
   </property>
   <property name="statusTip">
    <string>Fill holes in the segmentation</string>
   </property>
  </action>
  <action name="actionSmooth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme=":/images/postprocess_smooth.png"/>
   </property>
   <property name="text">
    <string>Smooth</string>
   </property>
   <property name="toolTip">
    <string>Smooth the segmentation</string>
   </property>
   <property name="statusTip">
    <string>Smooth the segmentation</string>
   </property>
  </action>
  <action name="actionEditSegmentation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Segmentation</string>
   </property>
  </action>
  <action name="actionEditContours">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Contours</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
